### ISSUE TITLE
Only converting first few minutes to mp3

### ISSUE THREAD

**Issue by halfbakedleek** [openaudible/openaudible#945]
Hi, I have a 15 hour long file for which only the first 20-21 minutes is getting converted when I convert from an aax file on my hard drive. It looks OA recognizes that it's 15 hours long (I can see the progress messages saying "Converting xyz of 15:xx:xx"), but after getting to 2-3% it just abruptly stops and reports success. Sure enough, when I transfer the file to my device and play it, it's only 21 minutes long. The file size also looks really small (19MB) compared to the aax file size (a few hundred MB).

**Comment by openaudible** [comment]
That is a strange one. Any error messages in the console window?

Can you try setting the format to m4b and trying to convert that? That would narrow down whether it is a mp3 error or something else.

**Comment by halfbakedleek** [comment]
I can't seem to repro it anymore for the same book if I download via syncing my account to audible. When I synced my audible account, downloaded directly from audible -> OpenAudible (as opposed to importing my previously downloaded aax files from Audible -> Hard Drive -> Open Audible via File -> Import Books), I no longer have the issue. When I tried to import a file directly from the hdd, I did get the error. Now even if I try to 'go back' and download the file and import via file, I can't repro it. I had tried both mp3 and m4b yesterday and both had the same issue.
